printer.go

```diff
@@ -58,6 +92,36 @@ func Default() (string, error) {
 	return syscall.UTF16ToString(b), nil
 }
 
+func GetDefaultPrinterType() (string, error) {
+	printerName, _ := Default()
+	return GetPrinterType(printerName)
+}
+
+func GetPrinterType(printerName string) (string, error) {
```

Comment:
Maybe instead of returning sting, this function should return more info. Maybe create
```
type DriverInfo struct {
...
Attributes uint32
}
```
, then you could do:
```
di, err := myprinter.DriverInfo()
if err != nil {
...
}
if di.Attributes&PRINTER_DRIVER_XPS != 0 {
...
}
```
We could start DriverInfo struct just with Attributes field and add more fields later as needed.

Reply:
Ok, i made the Driver info with driver name, enviroment and driver path